projects/plugins/jetpack/modules/publicize/publicize-jetpack.php

```diff
@@ -448,25 +614,37 @@ public function post_is_done_sharing( $post_id = null ) {
 			return false;
 		}
 
-		return 'publish' == $post->post_status || get_post_meta( $post->ID, $this->POST_DONE . 'all', true );
+		return 'publish' === $post->post_status || get_post_meta( $post->ID, $this->POST_DONE . 'all', true ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
 	}
 
 	/**
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
+	 *
+	 * @param int      $post_ID Post ID.
+	 * @param \WP_Post $post Post object.
 	 */
-	function save_publicized( $post_ID, $post = null, $update = null ) {
+	public function save_publicized( $post_ID, $post = null ) {
 		if ( is_null( $post ) ) {
 			return;
 		}
-		// Only do this when a post transitions to being published
+		// Only do this when a post transitions to being published.
+		// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
I'm not sure if it's worth doing this for this change, but should we make these constants so it doesn't complain about the naming?

Reply:
We probably should, but I'd prefer to hold off for now and change it as part of the bigger structural improvements. My concern is other (third-party) code using these constants. Any idea if that's even a realistic concern?

Reply:
Mmmm, it's unlikely, but if we made it them static class members, then I think we could use this naming convention and you it wouldn't actually break, you'd just get a warning that you're accessing a static property as an instance one. 

Having said that, I think you're right that we shouldn't take this on right now.